### PR TITLE
[US207 Task309] Creating pipeline

### PR DESCRIPTION
**Pull request by achaur11**

### CODE REVIEW

**Comment by shahih25** [comment]
Why aren't we running the python tests?

File: burndown-chart/setup.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from setuptools import find_packages, setup
+
+setup(
+    name='curndown-chart',
```
Comment (shahih25) [inline burndown-chart/setup.py]: burndown

Comment (achaur11) [reply burndown-chart/setup.py]: Wow. Fixed.

**Review by shahih25 — APPROVED** [review]
lgtm

**Pull request merged**